packages/react-components/react-drawer/library/src/components/OverlayDrawer/useOverlayDrawerStyles.styles.ts

```diff
@@ -95,17 +49,11 @@ export const useOverlayDrawerStyles_unstable = (state: OverlayDrawerState): Over
     baseClassNames,
     resetStyles,
     rootStyles[state.position],
-    drawerMotionClassNames,
     state.root.className,
   );
 
   if (backdrop) {
-    backdrop.className = mergeClasses(
-      overlayDrawerClassNames.backdrop,
-      backdropMotionClassNames,
-      durationStyles[state.size],
-      backdrop.className,
-    );
+    backdrop.className = mergeClasses(overlayDrawerClassNames.backdrop, backdrop.className);
```

Comment:
Not necessarily part of this PR, but:
Each Drawer size preset has a different animation duration, and the Backdrop animation should match this.
Is there a way to use the new Motion APIs on this slot so the duration matches the Drawer?

Reply:
Aha, good catch 👍